packages/manager-react-components/CHANGELOG.md

```diff
@@ -3,9 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [1.41.1](https://github.com/ovh/manager/compare/@ovh-ux/manager-react-components@1.41.0...@ovh-ux/manager-react-components@1.41.1) (2024-11-13)
+## [2.2.0](https://github.com/ovh/manager/commit/040237b6b436b9c1e5b76c9fed3076102aee3c1e) (2024-11-15)
 
-**Note:** Version bump only for package @ovh-ux/manager-react-components
+- Udpate ods version to 18.3.0
+- Move ods to peerDependencies
+- Update ODS version in Generator
+- Pre-build documentation
+
+# [2.1.0](https://github.com/ovh/manager/commit/377f4206c7e9a4f43aec27b25a1a617e62aa5dba) (2024-11-13)
+
+- Upgrade ods version 18
+- Update generator
+- Fix bump ods 18 issues
```

Comment:
Can you please mention the actual commit details like https://github.com/ovh/manager/blob/62239840182bd3c5c99a31b37d68731555396e31/packages/manager-react-components/CHANGELOG.md#1410-alpha9-2024-11-08? This makes it easy to look for the commit through the commit history.

Reply:
sure !